synthtool/py_templating_instructions/README.md

```diff
@@ -0,0 +1,32 @@
+# Using Synthtool for Python Samples
+
+Synthtool supports template files using [Jinja](http://jinja.pocoo.org/).
+
+Templates used for the Python samples are found in subdirectories of [`synthtool/gcp/templates/python_samples`](gcp/templates/python_samples)
```

Comment:
Maybe move these down or remove? They don't seem that important to anyone generating snippets